### DISCUSSION TITLE
Nerd miner stuck

### DISCUSSION THREAD

**Discussion by reinaldoie** [BitMaker-hub/NerdMiner_v2#500] [General]
Does anyone know how to fix this?![20240928_204647.jpg](https://github.com/user-attachments/assets/aefa6d96-6305-4f52-b212-3d6c01128c6b)

![ae_1727552765363.jpg](https://github.com/user-attachments/assets/cdba7a71-803e-41b5-83fd-15b3fabe4d41)

**Comment by sany3001** [comment]
If you mean that “0”, double check your BTC address. Also maybe check that your network can reach pool.

**Reply by reinaldoie** [reply]
Problem solved. Thanks for the help, I did exactly what you suggested and it solved the problem. Thanks again.
